.dockerignore

```diff
@@ -2,3 +2,17 @@
 deps/*
 doc
 ebin
+AUTHORS
+CONTRIBUTING
+CONTRIBUTORS
+build.cmd
+Dockerfile
+Dockerfile.*
+LICENSE
+Makefile
+prepare-release.sh
+*.md
+start.*
+TRANSLATORS
+zotonic_install
+priv/emacs
```

Comment:
There’s definitely something to be said for that more conservative viewpoint.

However, I still prefer a `.dockerignore` that using the negation pattern for three reasons:

1. One of the purposes of using Alpine is to bring down the image size as much as we can (under 194 MB now using negation).
2. The directory structure is relatively stable. Usually, files that are necessary for the build will end up in one of the already existing (and properly unignored) directories.
3. If some file turns up elsewhere (and it will sometime, of course), the Docker build failures will tell us clearly anyway.


I’ll switch to negation for now; if this turns out to be unmaintainable, I’ll revert back to your solution.